utilities/decrypt_data/export_postgres_to_excel_chunks.py

```
@@ -0,0 +1,585 @@
+        """Close database connection."""
+        if getattr(self, 'engine', None) is not None:
+            try:
+                self.engine.dispose()
+            except Exception:
+                pass
+            self.engine = None
+        if self.connection:
+            self.connection.close()
+            logger.info("Database connection closed")
+    
+    def get_all_tables(self, schema: str = 'public') -> List[str]:
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)